### ISSUE TITLE
New cron entry: run daily adif backup

### ISSUE THREAD

**Issue by hirschnase** [Maescool/docker-Cloudlog#4]
Hello,

I'd like to suggest to add this line into cron-cloudlog

```@daily curl --silent http://localhost/index.php/backup/adif &>/dev/null```

This call generates an Adif backup of all QSOs in the backup-directory of Cloudlog.

**Comment by Wireheadbe** [comment]
I've noticed cron doesn't do anything at all - because cron is started in a different context from web, and thus http://localhost doesn't work. 
It would probably work if we point to http://web/somethingsomething - that will make it work. I'll test and make a pull request.

**Comment by Wireheadbe** [comment]
Yup, this does the trick. Backup via the docker instance is a no-go at the moment. Because these days it requires the API key to execute a backup, and that is not available when you're bootstrapping your instance. Manual for now, but at least the existing entries will work with 
```
--- a/docker/cron-cloudlog
+++ b/docker/cron-cloudlog
@@ -1,3 +1,3 @@
-@weekly curl --silent http://localhost/index.php/lotw/load_users &>/dev/null
-@weekly curl --silent http://localhost/index.php/update/dxcc &>/dev/null
-@weekly curl --silent http://localhost/index.php/update/update_clublog_scp &>/dev/null
+@weekly curl --silent http://web/index.php/lotw/load_users &>/dev/null
+@weekly curl --silent http://web/index.php/update/dxcc &>/dev/null
+@weekly curl --silent http://web/index.php/update/update_clublog_scp &>/dev/null
```
Filed https://github.com/Maescool/docker-Cloudlog/pull/12 - but automated backup will most-likely not be in the possible scope of the docker container.
